app-mobilephone/scrcpy/scrcpy-2.1.1.ebuild

```diff
@@ -23,7 +23,7 @@ DEPEND="media-libs/libsdl2[X]
 # pipewire workaround see https://github.com/Genymobile/scrcpy/issues/3864
 RDEPEND="${DEPEND}
 	!ppc64? ( dev-util/android-tools )
-	pipewire? ( media-libs/libsdl2[-alsa,pipewire] )"
```

Comment:
To me, it looks like we should drop this entirely. scrcpy has no business here. At best, we could ewarn about it.

Reply:
You right and this workaround cause [some problem](https://forums.gentoo.org/viewtopic-t-1164347.html).
A message like
```
If you use pipewire because of a problem with libsdl2 it is possible that 
scrcpy will not start, in which case start the program by exporting the 
environment variable SDL_AUDIODRIVER=pipewire.
For more information see https://github.com/Genymobile/scrcpy/issues/3864.
```
Can this be enough?

Reply:
sounds good to me